superset-frontend/src/components/AlteredSliceTag/index.tsx

```diff
@@ -98,48 +94,83 @@ function alterForComparison(value?: string | null | []): string | null {
     return null;
   }
   return value;
-}
-
-class AlteredSliceTag extends React.Component<
-  AlteredSliceTagProps,
-  AlteredSliceTagState
-> {
-  constructor(props: AlteredSliceTagProps) {
-    super(props);
-    const diffs = this.getDiffs(props);
-    const controlsMap: ControlMap = getControlsForVizType(
-      props.origFormData.viz_type,
-    ) as ControlMap;
-    const rows = this.getRowsFromDiffs(diffs, controlsMap);
+};
 
-    this.state = { rows, hasDiffs: !isEmpty(diffs), controlsMap };
+export const formatValueHandler = (
+  value: any,
```

Comment:
Why did we change the type to `any`?

Reply:
an oversight from the AI changes, meshing a number of different file revisions from different sources